packages/core/src/textures/Texture.ts

```diff
@@ -309,7 +309,13 @@ export class Texture extends EventEmitter
      */
     clone(): Texture
     {
-        return new Texture(this.baseTexture, this.frame, this.orig, this.trim, this.rotate, this.defaultAnchor);
+        return new Texture(this.baseTexture,
+            this.frame.clone(),
+            this.orig.clone(),
+            this.trim.clone(),
+            this.rotate,
+            this.defaultAnchor
```

Comment:
Are we ok with the defaultAnchor copied by reference?

Reply:
I think trim could be undefined or null perhaps and needs check

Reply:
defaultAnchor is cloned in the constructor anyways

Reply:
I added another test check for `defaultAnchor` to make sure it isn't the same reference. Also, made `trim` optional (thanks @EneaEntertainment)